frontend/packages/kubevirt-plugin/integration-tests/deploy-kubevirt-gating.sh

```diff
@@ -54,35 +120,31 @@ oc create -f https://raw.githubusercontent.com/kubevirt/hyperconverged-cluster-o
 # Wait for kubevirt virt-operator to be available
 oc -n kubevirt-hyperconverged wait deployment/virt-operator --for=condition=Available --timeout="10m"
 
-# -----------------
+# ----------------------------------------------------------------------------------------------------
 # Create storage class and storage namespace for testing
+# Install HPP
 
-# Create openshift-cnv namespace for Integration Tests
-oc create namespace openshift-cnv
+export HPP_VERSION="release-v0.8"
 
-# Setup storage
-oc create -f https://raw.githubusercontent.com/openshift/console/master/frontend/packages/kubevirt-plugin/integration-tests/ci-scripts/hostpath-provisioner-setup.yml
+# Configure SELinux when using OpenShift, will allow hpp to create storage on workers.
```

Comment:
nitpick: Change to "Configure SELinux when using OpenShift to allow HPP to create storage on workers"